resources/assets/less/bem/difficulty-badge.less

```diff
@@ -17,3 +16,7 @@
     margin-right: 4px;
   }
 }
+
+.difficulty-badge--expert-plus {
+  color: @osu-colour-orange-1
+}
```

Comment:
what I was saying is instead of

```less
@less-var-1: value;

.class {
  --css-var-1: @less-var-1;
}
```
it should be just
```css
.class {
  --css-var-1: value;
}
```
or alternatively a complementing `getDiffContentColour` function.

(also nothing else create modifier class explicitly like this; usually just `&--modifier` inside the class)

Reply:
That variable is declared elsewhere.

https://github.com/ppy/osu-web/blob/769a95d29e4f38ad0dc7537d1696c58e651e2cf4/resources/assets/less/colors.less#L173